redisinsight/ui/src/pages/browser/components/key-details-header/components/Formatter/styles.module.scss

```diff
@@ -15,6 +15,31 @@
   :global(.euiFormControlLayout) {
     height: 100%;
   }
+
+  width: 92px;
```

Comment:
cloud you move these styles before nested: `:global(...)`?

Reply:
done